### PR TITLE
bump implementations in build.gradle

### PR DESCRIPTION
**Pull request by Jvr2022**
bump com.google.android.material:material to 1.11.0 bump androidx.browser:browser to 1.7.0
bump androidx.work:work-runtime-ktx to 2.9.0
bump org.apache.commons:commons-lang3 to 3.14.0
bump org.jsoup:jsoup to 1.17.1
bump com.github.skydoves:balloon to 1.6.3

### CODE REVIEW

**Comment by cooltey** [comment]
Please make sure this passes the test.

**Comment by Jvr2022** [comment]
> Please make sure this passes the test.

i am gonna fix itt

**Comment by Jvr2022** [comment]
ready for review

File: app/build.gradle [hunk 1]
```diff
@@ -183,10 +183,10 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinCoroutinesVersion"
     implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:$serialization_version"
 
-    implementation "com.google.android.material:material:1.10.0"
+    implementation "com.google.android.material:material:1.11.0-rc01"
```
[the remark this answers is not in the document]
Comment (Jvr2022) [reply app/build.gradle:186]: I am gonna change that today

**Pull request merged**